packages/components/card/src/base-card/BaseCard.styles.ts

```diff
@@ -40,6 +40,12 @@ export const getBaseCardStyles = () => {
         textDecoration: 'none',
         transition: `border-color ${tokens.transitionDurationDefault} ${tokens.transitionEasingDefault},
     box-shadow ${tokens.transitionDurationShort} ${tokens.transitionEasingDefault}`,
+
+        '&:focus': css({
```

Comment:
shouldn't we do the same for `&:hover`?

Reply:
hover is actually depending on the fact if the card is interactive or not, so if it's a button. Maybe it's easier to keep it as it is in that case?